### PR TITLE
3 header

### PR DESCRIPTION
**Pull request by thegta5p**
RtoolsIssue3

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,5 +1,10 @@
 const Header = ({ text }) => {
-  return <div>Header</div>;
+  return (
+    <>
+      <h1 className="text-white">{text}</h1>
```
Comment (menthy-wu) [inline src/components/Header.jsx:4]: use <p> tag, and give a font size

Comment (thegta5p) [reply src/components/Header.jsx]: Added in missing stuff.

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,5 +1,10 @@
 const Header = ({ text }) => {
-  return <div>Header</div>;
+  return (
+    <div>
+      <h1 className="text-white text-4xl mb-0">{text}</h1>
```
Comment (menthy-wu) [inline src/components/Header.jsx:4]: use <p> tag instead of <h1>

Comment (thegta5p) [reply src/components/Header.jsx:4]: By tag do you mean change h1 to div?

Comment (menthy-wu) [reply src/components/Header.jsx:4]: to <p> 
https://www.w3schools.com/tags/tag_p.asp

Comment (menthy-wu) [reply src/components/Header.jsx:4]: sorry about the confusion

Comment (thegta5p) [reply src/components/Header.jsx]: Alright i fixed it

File: src/components/Header.jsx [hunk 3]
```diff
@@ -1,5 +1,10 @@
 const Header = ({ text }) => {
-  return <div>Header</div>;
+  return (
+    <div>
+      <p className="text-white text-4xl mb-0">{text}</p>
+      <div className="bg-green-300 rounded h-1 w-20"></div>
```
Comment (menthy-wu) [inline src/components/Header.jsx:5]: change the div tag to self closing so something like this \<div/>

Comment (thegta5p) [reply src/components/Header.jsx]: Fixed it

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**